### PR TITLE
add sample config for TS search; rename to match TS playbook

### PR DESCRIPTION
**Pull request by martenson**

### CODE REVIEW

File: config/tool_shed.ini.sample [hunk 1]
```diff
@@ -46,6 +46,14 @@ session_data_dir = %(here)s/database/beaker_sessions
 session_key = galaxysessions
 session_secret = changethisinproduction
 
+# -- Repository and Tool search
+# Using the script located at scripts/build_ts_whoosh_index.py
+# you can generate search index and allow full text API searching over
+# the repositories and tools within the Tool Shed given that you specify
+# the following two config options.
+toolshed_search_on = True
+whoosh_index_dir = database/toolshed_whoosh_indexes
```
Comment (jmchilton) [inline config/tool_shed.ini.sample]: I think these actual options should be commented out in the sample file if reasonable defaults are possible (which they are).

Comment (martenson) [inline config/tool_shed.ini.sample]: I changed it - still unclear to me where in config it should be set/commented/loaded/defaulted but I don't want to get into that debate again.

Comment (jmchilton) [inline config/tool_shed.ini.sample]: :) I think that other... exchange... was a different debate. The goal for the .sample files is to describe the default with comments as Galaxy will load it without actually setting any values that can be defaulted.

**Comment by jmchilton** [comment]
Thanks for the change. :+1:

File: config/tool_shed.ini.sample (hunk 1 above)
Comment (martenson) [inline config/tool_shed.ini.sample]: we have:
`use_new_layout = true`
`require_login = False`
`new_file_path = database/tmp`
and multiple others that do not fit in that description imo.

Comment (jmchilton) [inline config/tool_shed.ini.sample]: Each is a doc bug IMO.

File: scripts/tool_shed/build_ts_whoosh_index.py [hunk 2]
```diff
@@ -6,7 +6,9 @@
 
 Make sure you adjusted your config to:
  * turn on searching via toolshed_search_on
- * specify toolshed_whoosh_index_dir where the indexes will be placed
+ * specify whoosh_index_dir where the indexes will be placed
+
+Also make sure that GALAXY_EGGS_PATH variable is properly set.
```
Comment (dannon) [inline scripts/tool_shed/build_ts_whoosh_index.py:11]: It works without setting GALAXY_EGGS_PATH at all if invoked as above from the root, right?

Comment (martenson) [inline scripts/tool_shed/build_ts_whoosh_index.py:11]: Yeah it runs from the box, but if you set your GALAXY_EGGS_PATH for galaxy you have to set it for this script too.

Comment (dannon) [inline scripts/tool_shed/build_ts_whoosh_index.py:11]: Ok, perfect, this looks good to me.  Thanks!

**Comment by dannon** [comment]
+1

**Pull request merged**